terraform/gke.tf

```diff
@@ -27,7 +27,7 @@ module "gke" {
   enable_shielded_nodes        = true
   grant_registry_access        = true
   gcs_fuse_csi_driver          = true
-  http_load_balancing          = false
+  http_load_balancing          = true
```

Comment:
Why do we need this?

Reply:
This is needed by ConfigSync that refuse to install the CRD if the HTTP Load Balancing is not enable on the cluster